site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Shadow endpoints and update detection come to SvelteKit"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Some long-requested features have been added to SvelteKit this month... including shadow endpoints! This change in how `load` functions makes it easier to fetch data required for basic pages, redirect from POST responses and handle 404s and other errors.
+
+More on that and other new features and fixes below!
+
+## New across Svelte and Language Tools
+- Any highlights from the Svelte Changelog? Feels like it was mostly bugfixes
+...
+- svelte.dev is [now available in Russian](https://ru.svelte.dev/)! If you'd like to contribute to the community transaltions, check out [sveltejs-translations/content-server](https://github.com/sveltejs-translations/content-server)
```

Comment:
I think it existed for quite some time already, but they adjusted their workflow to mimic the docs API that Svelte currently uses instead - but that API might not be for long anymore, all in flux honestly. So not sure how we could rephrase that.

Reply:
Yes the Russian website has been around for a very long time, the new thing is the API, as @dummdidumm mentioned here

Reply:
Going to take this out, then.